Fix/modify integration slider

bsunnquist opened the pull request:
This PR fixes the issue of the integration slider not working on preview images. It also adds the total ints to the output.

pep8speaks commented:
Hello @bsunnquist, Thank you for submitting the Pull Request !

* In the file [`jwql/website/apps/jwql/data_containers.py`](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py):

> [Line 59:1](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L59): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 68:1](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L68): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 69:1](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L69): [E402](https://duckduckgo.com/?q=pep8%20E402) module level import not at top of file
> [Line 366:41](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L366): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 597:5](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L597): [E115](https://duckduckgo.com/?q=pep8%20E115) expected an indented block (comment)
> [Line 597:5](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L597): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 924:13](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L924): [E125](https://duckduckgo.com/?q=pep8%20E125) continuation line with same indent as next logical line
> [Line 925:17](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L925): [E117](https://duckduckgo.com/?q=pep8%20E117) over-indented
> [Line 963:1](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L963): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 1092:31](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L1092): [E261](https://duckduckgo.com/?q=pep8%20E261) at least two spaces before inline comment
> [Line 1185:9](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L1185): [E722](https://duckduckgo.com/?q=pep8%20E722) do not use bare 'except'
> [Line 1273:21](https://github.com/spacetelescope/jwql/blob/1eb6fccacb3b7cca108f4b8054920825d47986db/jwql/website/apps/jwql/data_containers.py#L1273): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent

If you have not done so, please consult the [`jwql` Style Guide](https://github.com/spacetelescope/jwql/blob/main/style_guide/README.md)

bsunnquist commented:
This addressed points 1 and 2 in https://github.com/spacetelescope/jwql/issues/873

This slider also had a bug for images with >10 ints, so I fixed that too

bsunnquist commented:
@bourque this is ready for review whenever

bsunnquist commented:
@bourque ok, it should work now no matter how many pngs/ints there are (and should also still work if we increase that ratio in the future). The only thing I couldn't get working quite right was the live update of the slider number as you drag it.
